pennylane/data/data_manager/__init__.py

```diff
@@ -361,110 +356,54 @@ def load(  # pylint: disable=too-many-arguments
     return [Dataset.open(path, "a") for path in download_paths]
 
 
-def list_datasets() -> dict:
-    r"""Returns a dictionary of the available datasets.
-
-    Return:
-        dict: Nested dictionary representing the directory structure of the hosted datasets.
-
-    .. seealso:: :func:`~.load_interactive`, :func:`~.list_attributes`, :func:`~.load`.
-
-    **Example:**
-
-    Note that the results of calling this function may differ from this example as more datasets
-    are added. For updates on available data see the `datasets website <https://pennylane.ai/datasets>`_.
+def _interactive_request_data_name(data_names):
+    """Prompt the user to select a data name."""
+    print("Please select the data name from the following:")
+    for i, option in enumerate(data_names):
+        print(f"{i + 1}: {option}")
+    choice = input("Choice of data name: ")
+    if choice not in data_names:
+        raise ValueError(f"Must select a single data name from {data_names}")
+    return choice
 
-    >>> available_data = qml.data.list_datasets()
-    >>> available_data.keys()
-    dict_keys(["qspin", "qchem"])
-    >>> available_data["qchem"].keys()
-    dict_keys(["H2", "LiH", ...])
-    >>> available_data['qchem']['H2'].keys()
-    dict_keys(["CC-PVDZ", "6-31G", "STO-3G"])
-    >>> print(available_data['qchem']['H2']['STO-3G'])
-    ["0.5", "0.54", "0.62", "0.66", ...]
 
-    Note that this example limits the results of the function calls for
-    clarity and that as more data becomes available, the results of these
-    function calls will change.
-    """
-
-    def remove_paths(foldermap):
-        """Copies the foldermap, converting the bottom-level mapping of parameters
-        to Paths to a list of the parameters."""
-        value = next(iter(foldermap.values()))
-
-        if not isinstance(value, Mapping):
-            return sorted(foldermap.keys())
-
-        return {param: remove_paths(foldermap[param]) for param in foldermap.keys()}
-
-    return remove_paths(_get_foldermap())
-
-
-def list_attributes(data_name):
-    r"""List the attributes that exist for a specific ``data_name``.
-
-    Args:
-        data_name (str): The type of the desired data
+def _interactive_request_attributes(attribute_options):
+    """Prompt the user to select a list of attributes."""
+    print(
+        'Please select a list of attributes from the following available attributes or "full" for all attributes.'
+    )
+    for i, option in enumerate(attribute_options):
+        print(f"{i + 1}: {option}")
+    choice = input("Choice of attributes: ")
```

Comment:
```suggestion
    choice = input("Choice of attributes: ").strip()
```

Reply:
This can be either a list or a string, logic updated accordingly to strip whitespace for either case